app/i18n/locales/zh-CN.js

```diff
@@ -269,6 +269,10 @@ export default {
 	SAVE: '保存',
 	Save_Changes: '保存更改',
 	Save: '保存',
+	Save_To_Camera_Error: '错误',
+	Save_To_Camera_Error_Message: '你拒绝访问相机胶卷！',
+	Save_To_Camera_Success: '成功',
+	Save_To_Camera_Success_Message: '照片添加到相机胶卷！',
```

Comment:
````
Save_To_Camera_Error_Message: '你已拒绝访问图库！',
...
Save_To_Camera_Success_Message: '图片已保存至图库！',
````
The above expression seems more in line with Chinese habits. Please change them, thx :)